lib/src/components/modal.dart

```diff
@@ -101,14 +105,7 @@ class MDModal extends StatelessWidget {
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
               header,
-              const SizedBox(height: 22.0),
-              SingleChildScrollView(
-                child: Padding(
-                  padding: const EdgeInsets.all(16.0),
-                  child: body,
-                ),
-              ),
-              const SizedBox(height: 16.0),
+              Expanded(child: Padding(padding: bodyPadding, child: SingleChildScrollView(child: body))),
```

Comment:
we shouldn't do this, the modal will take the whole screen, even if we have just one line to show there